manga compiling script.py

```diff
@@ -1,92 +0,0 @@
-import os
-import shutil
-import tempfile
-from pathlib import Path
-
-final_dir_name = str(input('What is the name and volume of the manga you '
-                         'are compiling (ex: chainsaw man volume 1): '))
-
-# final_dir_name = final_dir_name.replace(" ", "-")
-# final_dir_name = "Chainsaw Man Volume " + final_dir_name
-
-# get basic download path to access downloaded files
-downloads_path = str(Path.home() / "Downloads")
-
-# make new directory path in downloads from given name and volume
-final_dir_path = os.path.join(downloads_path, final_dir_name)
-
-# makes a new directory from new path - kinda unnecessary
-os.mkdir(final_dir_path)
-
-
-# self-explanatory; choose what chapters you want in said volume
-chapter_start = int(input('What Chapter are you starting at: '))
-chapter_end = int(input('What Chapter are you ending at: '))
-
-# Give the general download name that can be iterated through for each chapter
-general_download_name = str(input("What is the name of the downloaded compressed files "
-                                  "including file type and a # symbol for the chapter number "
-                                  "(ex: chainsaw-man-#.zip): "))
-
-# general_download_name = "chainsaw-man-#.zip"
-
-gen_download_path = str(input('Enter the path to find the downloaded files: '))
-
-# Creates path to general download name
-# gen_download_path = os.path.join(downloads_path, general_download_name)
-
-gen_download_path = os.path.join(gen_download_path, general_download_name)

```

Comment:
If you're interested, you could make all of this a CLI and add the program as a command in your terminal. To run your program, you navigate to the directory with the files of interest in your terminal and input all the parameters at once. For example, if you added a command named `generate-cbz` to your terminal that runs this program, it could look something like this:

`$ generate-cbz <name> <volume> <chapter_start> <chapter_end>`

So an example command would look like:

`$ generate-cbz chainsaw_man 1 1 6`

which generates chainsaw man volume 1 from chapters 1 thru 6.

There's a couple of reasons you might want to do this:

1. It makes it easier to add features. If you wanted to add support for another file conversion, currently you would need to add more input prompts and/or rewrite existing ones and it could get ugly. With a CLI you just add another parameter to specify when you run, and it's easier to document how it works
2. Input prompts are only operable by humans, but a CLI is operable by humans and machines. What if you had a file of 200 zips and you wanted to create 15 different volumes from it in one go? With a CLI you could write a program that does this. (I don't know that you would ever want to do this)
3. (subjective) it's easier to use. One input, one output.

Reply:
I've been looking into creating a gui for this program to do something along those lines. Right now i just want it to convert png to cbz cuz pdf quality is always lower than when i can get the raw pngs. Not sure if the CLI feature would work the way that you gave an example command because that doesn't specify the directory where the files are or the current file names. Right now I'm trying to figure out a way to select multiple files from file explorer and be able to 1) automatically sort them in ascending order based on a naming pattern or 2) sort them with a drag and drop function.